spec/firebase_device_spec.rb

```diff
@@ -0,0 +1,97 @@
+require 'spec_helper'
+
+LOCALE_SAMPLE_DATA_PATH = File.join(__dir__, 'test-data', 'firebase', 'firebase-locale-list.json')
+MODEL_SAMPLE_DATA_PATH = File.join(__dir__, 'test-data', 'firebase', 'firebase-model-list.json')
+VERSION_SAMPLE_DATA_PATH = File.join(__dir__, 'test-data', 'firebase', 'firebase-version-list.json')
```

Comment:
It's usually not a good practice to define constants like that in Unit Tests — because those constants have visibility in the whole gem. I'd suggest you to convert them all to `let(:name) { value }` inside the top-level `describe` instead.

Reply:
This is a bit weird, because I want to share them between all tests – I adjusted it in https://github.com/wordpress-mobile/release-toolkit/commit/5d15b3a8a8defa7b83068034e59a3491b92f14c9, interested to hear your thoughts?

Reply:
Moved these over in 85b0df093ab6e5491346b06d24375235ff1f3d07